model/src/main/kotlin/DependencyGraphNavigator.kt

```diff
@@ -127,7 +127,7 @@ private class DependencyRefCursor(
     val graph: DependencyGraph,
 
     /** The [DependencyGraphNode]s to traverse. */
-    val nodes: Collection<DependencyGraphNode> = emptyList(),
+    nodes: Collection<DependencyGraphNode> = emptyList(),
```

Comment:
Nit: "Drop" sounds as if it was removed completely. Maybe better say "Turn a property into a plain constructor argument".